services/discordService.js

```diff
@@ -51,6 +51,20 @@ const addRoleToUser = async (userid, roleid) => {
   return response;
 };
 
+const setUserDiscordNickname = async (userName, discordId) => {
+  const authToken = await generateAuthTokenForCloudflare();
```

Comment:
It should be in a try-catch block ideally

Reply:
+1